app/controllers/admin/api/registry/policies_controller.rb

```diff
@@ -7,7 +7,23 @@ class Admin::Api::Registry::PoliciesController < Admin::Api::BaseController
   representer ::Policy
 
 
-  # TODO: ApiDocs documentation :)
+  # swagger
+  ##~ sapi = source2swagger.namespace("Account Management API")
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/registry/policies.json"
+  ##~ e.responseClass = "policy"
+  #
+  ##~ op            = e.operations.add
+  ##~ op.httpMethod = "POST"
+  ##~ op.summary    = "APIcast Policies Registry Create"
```

Comment:
I would say `Gateway Policies Registry Create` More generic. Remember we also have service mesh integration.